### PR TITLE
[12.0][MIG] account_check_printing_report_dlt103

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

Depends on 
- [x] #241 

CC @Eficent

### CODE REVIEW

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM code review

File: account_check_printing_report_dlt103/report/check_print.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class ReportCheckPrint(models.AbstractModel):
+    _name = 'report.account_check_printing_report_dlt103.report_check_dlt103'
+    _inherit = 'report.account_check_printing_report_base.report_check_base'
+
+    @api.multi
```
Comment (MiquelRForgeFlow) [inline account_check_printing_report_dlt103/report/check_print.py]:
```suggestion
    @api.model
```

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Tested on runbot :+1:

**Comment by patel1008b** [comment]
@hveficent @mreficent @hveficent @jbeficent will it merge in main branch ?

**Comment by HviorForgeFlow** [comment]
@bhaveshselarka reviews are welcome

**Pull request merged**